### DISCUSSION TITLE
Negative Labels (in Skweak)

### DISCUSSION THREAD

**Discussion by SandroSteinwand** [explosion/spaCy#11485] [Help: Other Questions] [answered]
I am using skweak for labelling of certain entities in free text.

Is there any possibility to prohibit spacy from labelling certain tokens? Meaning, that certain constructs are hard to exclude when writing labelling functions but can never be the correct target.

It is quite cumbersome and ineffective to do that within single labelling functions.

**Comment by polm** [comment, marked as answer]
Sorry, we don't develop skweak, so it's probably best to ask directly at their repo. 

Based on docs [here](https://github.com/NorskRegnesentral/skweak/wiki/Step-2:-Aggregation#underspecified-labels) it looks like they do have some kind of mechanism for this.
